test/check_inputs.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pandas as pd
+
+def create_df(metadata, df_name=None):
```

Comment:
We could also support reading the metadata.pkl files that we have for each folder.

Reply:
okay will do and make another push